Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -555,3 +568,55 @@ void ABenchmarkGymGameModeBase::ReportAuthoritativePlayers_Implementation(const
 		}
 	}
 }
+
+void ABenchmarkGymGameModeBase::TickActorMigration(float DeltaSeconds)
+{
+	// Count how many actors hand over authority in 1 tick
+	int MovementAuthActors = GetMovementAuthActors();
+	int Delta = FMath::Abs(MovementAuthActors - PreviousTickMigration);
```

Comment:
The first time this is called the PreviousTickMigration will be zero... in fact, any time players connect they will cause the MovementAuthActors to increment.. as long as we ensure we don't check this metric too early I think it's fine.

Reply:
I use `ActorMigrationCheckDelay` for it